trunk/SUAVE/Methods/Aerodynamics/AVL/read_results.py

```diff
@@ -13,34 +14,91 @@
 def read_results(avl_object):
 
     results = Data()
-
+    i = 0
```

Comment:
Yes. It is currently not being used. However it is used in the code that has been commented out (for dynamic stability)